sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -662,7 +661,62 @@ Exceptions:
 
 |===
 
-Table 8. Member functions of the `command_graph` class (executable graph update).
+Table 8. Member functions of the `command_graph` class for queue recording.
+[cols="2a,a"]
+|===
+|Member function|Description
+
+|
+[source, c++]
+----
+using namespace ext::oneapi::experimental;
+bool begin_recording(const std::vector<queue>& queues)
```

Comment:
@sommerlukas Tagging for your thoughts on this PR, there is still some difference after this change between approaches:

1) `fusion_wrapper` takes a queue on construction, whereas a graph takes a
queue on `begin_recording()` rather than construction. I felt this was more suitable
give that when a graph is constructed, a user could want to use the explicit API for building up the API instead.

2) We pass a list of queues here and on `end_recording()`. I feel like if this extension was using fusion technology under the hood, and this created problems, then we could just omit doing the optimization in those circumstances.

Reply:
> Secondly, the fusion API allows to start and cancel/complete fusion on the same queue with different fusion_wrapper

That's interesting, it hadn't occurred to me that would be a valid usage. Sounds like there isn't anything we need to change here for now

Reply:
Yes, I agree, no need to change anything here. I only wanted to illustrate the motivation in our case, which differs from yours.